interface/weno/jsonScript.php

```diff
@@ -0,0 +1,132 @@
+    $wenoProvId = $proData[1]['weno_prov_id'];
+    $name       = $proData[1]['name'];
+    $phone      = str_replace("-","",$proData[1]['phone']);
+    $fax        = str_replace("-","",$proData[1]['fax']);
+    $street     = $proData[1]['street'];
+    $city       = $proData[1]['city'];
+    $state      = $proData[1]['state'];
+    $postalCode = $proData[1]['postal_code'];
+    $wenoAccId  = $proData[0][0]['gl_value'];
+    $wenoPass   = $proData[0][1]['gl_value'];
+
+function fillJsonProvider($fname,$lname,$npi,$wenoProvId,$name,$phone,$fax,$street,$city,$state,$postalCode,$wenoAccId,$wenoPass){
+	
+ $provider ='{"provider": {"provlname" : "'.$fname.'","provfname" : "'.$lname.'","provnpi" : '.$npi.',"facilityfax" : '.$fax.',"facilityphone" : '.$phone.',"facilityname" : "'.$name.'","facilitystreet" : "'.$street.'","facilitycity" : "'.$city.'","facilitystate" : "'.$state.'","facilityzip" : '.$postalCode.',"qualifier" : "'.$wenoProvId.'","wenoAccountId" : "'.$wenoAccId.'","wenoAccountPass" : "'.$wenoPass.'","wenoClinicId" : "'.$wenoProvId.'"}},';
+	return $provider; 
+}
+
+$pharmData = $prInfo->findPharmacy($id);
+
+$storename     = $pharmData[0]['name'];
+$storenpi      = $pharmData[0]['npi'];
+$pharmacy      = $pharmData[0]['ncpdp'];
+$pharmacyPhone = $pharmData[1][0]['area_code'].$pharmData[1][0]['prefix'].$pharmData[1][0]['number'];
+$pharmacyFax   = $pharmData[1][1]['area_code'].$pharmData[1][1]['prefix'].$pharmData[1][1]['number'];
+
+function fillJsonPharmacy($storename,$storenpi,$pharmacy,$pharmacyPhone,$pharmacyFax){
+	
+$pharmacy ='{"pharmacy": {"storename" : "'.$storename.'","storenpi" : '.$storenpi.',"pharmacy" : '.$pharmacy.',"pharmacyPhone" : '.$pharmacyPhone.',"pharmacyFax" : '.$pharmacyFax.'}},';
+	return $pharmacy;
+}	
+
+$drugData = $prInfo->oneDrug($data);
+
+    $drugName     = trim($drugData['drug']);
+    $drugNDC      = $drugData['drug_id'];
+    $dateAdded    = $drugData['date_Added'];
+    $quantity     = $drugData['quantity'];
+    $refills      = $drugData['refills'];
+    $dateModified = $drugData['date_Modified'];
+    $note         = $drugData['note'];
+    $take         = $drugData['dosage'];
+
+function fillJsonScript($drugName,$drugNDC,$dateAdded,$quantity,$refills,$dateModified,$note,$take){
+$script ='{"script": {"drugName" : "'.$drugName.'","drug_NDC" : "'.$drugNDC.'","dateAdded" : "'.$dateAdded.'","quantity" : "'.$quantity.'","refills" : "'.$refills.'","dateModified" : "'.$dateModified.'","note" : "'.$note.'","take" : "'.$take.'"}}]';
+		
+		return $script;
+}
+
+$completeJson =  fillJsonPatient($pfname,$plname,$pstreet,$ppostalCode,$pcity,$DOB,$sex) . 
+                 fillJsonProvider($fname,$lname,$npi,$wenoProvId,$name,$phone,$fax,$street,$city,$state,$postalCode,$wenoAccId,$wenoPass) . 
+				 fillJsonPharmacy($storename,$storenpi,$pharmacy,$pharmacyPhone,$pharmacyFax) . 
+				 fillJsonScript($drugName,$drugNDC,$dateAdded,$quantity,$refills,$dateModified,$note,$take);
```

Comment:
Brady, it seems we are miss communicating. array_merge is not going to produce the needed script transmit a prescription. Please review my notes in the  ajax_sample_code.php. The jsonscript.php file builds lines 70 -   113 programmatically. If it does not look like the hard code JSON script, it will be rejected by the WENO server.  So using the array_merge is not going to produce the script needed.